dabapush/Writer/FacebookDBWriter.py

```diff
@@ -27,28 +37,20 @@ def persist(self):
         data = self.buffer
         self.buffer = []
 
-        self.facebook_initializer.fb_insert(data)
+        for _ in data:
+            self.facebook_initializer.fb_insert(_)
+
+        self.facebook_initializer.local_session.commit()
 
     def __del__(self):
         print("Session and Connection Terminated")
+        super().__del__()  # this triggers self.persits and must be called
 
 
 class FacebookDBWriterConfiguration(DBWriterConfiguration):
+    """Configuration for the FacebookDBWriter"""
+
     yaml_tag = "!dabapush:FacebookDBWriterConfiguration"
 
-    def __init__(
-        self,
-        name,
-        id=None,
-        chunk_size: int = 2000,
-        user: str = "postgres",
-        password: str = "password",
-        dbname: str = "public",
-        host: str = "localhost",
-        port: int = 5432,
-    ) -> None:
-        super().__init__(name, id, chunk_size, user, password, dbname, host, port)
-
-    def get_instance(self):
-        """ """
+    def get_instance(self) -> FacebookDBWriter:  # pylint: disable=W0221
```

Comment:
pylint directive surpresses a `arguments-differ`-warning that triggers here for unknown reasons.